roles/jump-server/tasks/main.yml

```diff
@@ -2,43 +2,43 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
-- name: jump-server - add host
+- name: Jump-server - add host
```

Comment:
The name as it was `jump-server` was to signify a variable. If it's not going to be used as such in the name description just replace with `Jump server` in general. The hyphen seems a bit odd otherwise.

Reply:
I'll restore the lower casing to be consistent with other variable reference changes.